core/trino-main/src/main/java/io/trino/security/AccessControl.java

```diff
@@ -429,6 +429,7 @@ void checkCanRevokeRoles(SecurityContext context,
 
     /**
      * Check if identity is allowed to set role for specified catalog.
+     * Note: this is not used for system roles.
```

Comment:
is this comment still accurate?

Reply:
Yes, but it is not relevant so I remove it.  Also, the `@param` below this is wrong.